aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/SyncProcessorTest.java

```diff
@@ -116,6 +140,91 @@ public void setup() throws AmplifyException {
             .build();
     }
 
+    /**
+     * During a base sync, there are a series of events that should be emitted.
+     * This test verifies that these events are published via Amplify Hub depending
+     * on actions takes for each available model.
+     * @throws DataStoreException Not expected.
+     */
+    @Test
+    public void dataStoreHubEventsTriggered() throws DataStoreException {
+        // Arrange - BEGIN
+        // We're only emitting events for Post and Blogger
+        int expectedModelCount = 2;
```

Comment:
To self document it, you could express this as `int expectedModelCount = Arrays.asList(EventObject1.class, EventObject2.class).size()`.

Reply:
That's a good idea.

Reply:
Done.